### ISSUE TITLE
feat: Issue creation: link to sprint

### ISSUE THREAD

**Issue by imsergiobernal** [gioboa/jira-plugin#55]
It would be nice if you can link a new issue to an existing current sprint

**Comment by gioboa** [comment]
The difficult for this feature is find the API that give you the _"list of sprints"_

**Comment by gioboa** [comment]
@imsergiobernal 
Could you send me `sprint` field JSON?
You did the same thing [here](https://github.com/gioboa/jira-plugin/issues/53#issuecomment-451400272)
Thanks :+1:

**Comment by gioboa** [comment]
I close it for the moment.

**Comment by caleb15** [comment]
This would be a nice feature to add. 

Here's my sprint field JSON:

```
                        "customfield_10019": {
                            "required": false,
                            "schema": {
                                "type": "array",
                                "items": "string",
                                "custom": "com.pyxis.greenhopper.jira:gh-sprint",
                                "customId": 10019
                            },
                            "name": "Sprint",
                            "key": "customfield_10019",
                            "hasDefaultValue": false,
                            "operations": [
                                "set"
                            ]
                        },
```

**Comment by gioboa** [comment]
Thanks @caleb15 , 
At the moment I'm working for manage all the fields.
I'm studying Jira logic for every field, it's a hard work.

**Comment by gioboa** [comment]
@caleb15 do you have sprint field in the issue creation screen? 
In my Jira environment this particular field is locked.

**Comment by caleb15** [comment]
@gioboa yep

**Comment by gioboa** [comment]
With this API I can retrieve all the sprints options.
https://xxx/rest/greenhopper/1.0/sprint/picker
In this field you have only the sprints relative to the project? There are some filters like only open sprints?

**Comment by caleb15** [comment]
https://pastebin.com/4dxG8YQM

The sprint input box in jira when you expand it has a "Suggestions" and a "All sprints" category

**Comment by gioboa** [comment]
Available in [v 0.14.0](https://github.com/gioboa/jira-plugin/releases/tag/0.14.0)

**Comment by caleb15** [comment]
it works. Thanks! :)
